lib/models/tags.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = TagModelFactory;
+
+TagModelFactory.$provide = 'Models.Tag';
+TagModelFactory.$inject = [
+    'Model',
+    '_',
+    'Assert',
+    'Validatable',
+    'anchor',
+    'Promise'
+];
+
+function TagModelFactory (Model, _, assert, Validatable, Anchor, Promise) {
```

Comment:
planning on using Promise later in here, or was this a "just in case" that didn't get used?